cmd/reva/app-tokens-list.go

```diff
@@ -0,0 +1,223 @@
+// Copyright 2018-2021 CERN
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// In applying this license, CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+package main
+
+import (
+	"encoding/gob"
+	"io"
+	"os"
+	"reflect"
+	"strings"
+	"time"
+
+	applications "github.com/cs3org/go-cs3apis/cs3/auth/applications/v1beta1"
+	authpv "github.com/cs3org/go-cs3apis/cs3/auth/provider/v1beta1"
+	rpc "github.com/cs3org/go-cs3apis/cs3/rpc/v1beta1"
+	types "github.com/cs3org/go-cs3apis/cs3/types/v1beta1"
+	scope "github.com/cs3org/reva/pkg/auth/scope"
+	"github.com/jedib0t/go-pretty/table"
+)
+
+type appTokenListOpts struct {
+	Long              bool
+	All               bool
+	OnlyExpired       bool
+	ApplicationFilter string
+	Label             string
+}
+
+var listOpts *appTokenListOpts = &appTokenListOpts{}
+
+func appTokensListCommand() *command {
+	cmd := newCommand("token-list")
+	cmd.Description = func() string { return "list all the application tokens" }
+	cmd.Usage = func() string { return "Usage: token-list [-flags]" }
+
+	cmd.BoolVar(&listOpts.Long, "long", false, "long listing")
```

Comment:
Let's not make the formatting command-specific, but rather as a common functionality for all the commands. So you can remove the formatting from here and work on https://github.com/cs3org/reva/issues/1417 later